### PR TITLE
Add Bittrium Fortress Zone

### PR DESCRIPTION
**Pull request by ThePythonGuy3**
``The MainLand of all of the existing bittrium civilizations, it once was the main source of what now are called "HardMode bosses", but once was one of the most prosperating civilizations in the entire world. The base has been bombed many times by the sharded soldier mechs, and now its a place of desolation. Your mission is to conquer this place, and make sure your base doesnt get destroyed.``

Dunno if its a good map, but its fun. Enjoy it ig. :D

### CODE REVIEW

File: content/zones/bittrium-fortress.hjson [hunk 1]
```diff
@@ -0,0 +1,27 @@
+name: Bittrium Fortress
+description: The MainLand of all of the existing bittrium civilizations, it once was the main source of what now are called "HardMode bosses", but once was one of the most prosperating civilizations in the entire world. The base has been bombed many times by the sharded soldier mechs, and now its a place of desolation. Your mission is to conquer this place, and make sure your base doesnt get destroyed.
+startingItems: [
+	copper/4000
+	lead/4000
+	bittrium/4000
+	titanium/500
+  thorium/500
```
Comment (ghost) [inline content/zones/bittrium-fortress.hjson]: Cursed indent.

Comment (ThePythonGuy3) [reply content/zones/bittrium-fortress.hjson]: yes ideed wtf

**Comment by ThePythonGuy3** [comment]
fixed

**Comment by ghost** [comment]
*Use bundle.properties yes.*

**Comment by ThePythonGuy3** [comment]
no u

**Comment by sk7725** [comment]
> _Use bundle.properties yes._

dew it

**Comment by ThePythonGuy3** [comment]
hhhh

**Comment by ThePythonGuy3** [comment]
@sk7725 done, now merge

**Comment by sk7725** [comment]
Mainland, Hardmode, doesn't

**Comment by sk7725** [comment]
~~mergn't~~

**Comment by ThePythonGuy3** [comment]
what¿

**Comment by ThePythonGuy3** [comment]
oh.

**Comment by ghost** [comment]
Name them Koruh.

**Comment by ThePythonGuy3** [comment]
@sk7725 @sk7725 @sk7725

**Comment by sk7725** [comment]
> Name them Koruh.

Although they did use a form of the Koruh language, they are not related to Koruh.

**Pull request merged**